### PR TITLE
Move echo to S3 🎉

### PR DESCRIPTION
**Pull request by pvinis**
## Follow up:
- [ ] make a `production` branch from `master`.
- [x] fixup https://releases.artsy.net/admin/projects/56 to have `master` instead of `s3` and `production` instead of `s3-production` for all stages and deploy strategies.
- [x] Do we need a way to force deploy, even if there are no json changes? 🤔
  No. Even if we do, it's easy enough to add a dummy flag.

### CODE REVIEW

File: scripts/deploy.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const envsub = require('envsub')
```
Comment (ashfurrow) [inline scripts/deploy.js:1]: Of course JS has a dependency for this already 😂

Comment (pvinis) [reply scripts/deploy.js:1]: I would use envsubst but I didn't want to do the iso date in bash, and json validation and minification is easier in js too, so there we are 😅.

File: scripts/prepare.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const fs = require('fs')
```
Comment (damassi) [inline scripts/prepare.js:1]: Missing prettier?

Comment (pvinis) [reply scripts/prepare.js:1]: I'm pretty sure it's there working but I'll check again.

what made you think it's missing? 🤔👀👀

Comment (damassi) [reply scripts/prepare.js:1]: code doesn't look like rest of codebase (big 8 space tabs)

Comment (pvinis) [reply scripts/prepare.js:1]: I'll check 👌.

Comment (pvinis) [reply scripts/prepare.js:1]: In the end, it was my default prettier settings. I added an rc file. Good catch, thanks!

**Comment by pvinis** [comment]
This is ready to go. I'll do the horizon changes now, and once this is merged I'll make the `production` branch.

File: README.md [hunk 3]
```diff
@@ -1,38 +1,19 @@
...
 ---
 
 * __State:__ production
-* __Production:__ [https://echo-api-production.herokuapp.com/](https://echo-api-production.herokuapp.com/) | [Web Interface](https://echo-web-production.herokuapp.com) | [Heroku](https://dashboard.heroku.com/apps/echo-api-production/resources)
+* __Staging:__ [https://echo.artsy.net/Echo.staging.json](https://echo.artsy.net/Echo.staging.json)
+* __Production:__ [https://echo.artsy.net/Echo.json](https://echo.artsy.net/Echo.json)
 * __Github:__ [https://github.com/artsy/echo/](https://github.com/artsy/echo/)
-* __CI:__ [Semaphore](https://semaphoreci.com/artsy-it/echo/); merged PRs to artsy/echo#master are automatically deployed to staging; production is manually deployed from Semaphore
-* __Point People:__ [@ashfurrow](https://github.com/ashfurrow)
+* __Point People:__ [@ashfurrow](https://github.com/ashfurrow), [@pvinis](https://github.com/pvinis)
```
Comment (ashfurrow) [inline README.md:14]: 🌟

**Review by ashfurrow — APPROVED** [review]
🎉 Looks great! Thanks again.

**Pull request merged**